.github/workflows/run_integration_test.yml

```diff
@@ -1,6 +1,15 @@
 name: Run tests to ensure we can compile across arrow versions
 
-on: [workflow_dispatch, push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - 'CHANGELOG.md'
+  pull_request:
```

Comment:
While we're at it, can we consider using [types](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request) to prevent CI from triggering when the PR description changes? The documentation states that
> By default, a workflow only runs when a `pull_request` event's activity `type` is `opened`, `synchronize`, or `reopened`. To trigger workflows by different activity types, use the `types` keyword. 

I believe we're interested in not triggering on the `edited` activity type, which would have been manually added somewhere, according to the above?

Reply:
nice good find - ill edit!

Reply:
ref for future me: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows